src/StyleCopAnalyzersDefault.ruleset

```diff
@@ -38,7 +38,7 @@
    
     <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers.ReadabilityRules">
         <Rule Id="SA1100"  Action="Warning" />          <!-- Do not prefix calls with base unless local implementation exists -->
-        <Rule Id="SA1101"  Action="Warning" />          <!-- Prefix local calls with this -->
+        <Rule Id="SA1101"  Action="None" />          <!-- Prefix local calls with this -->
```

Comment:
💅 `<!-- Prefix local calls with this -->`  should be aligned with the others

Reply:
Done. Please check.